Scripts/boundary.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+from scipy.spatial import Delaunay
+from shapely.geometry import Polygon
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+fig = plt.figure()
```

Comment:
Put this in main so its not hard to find (unless this is standard in python)

Reply:
Lol actually I forgot to remove those since we don't need them.